doc/architecture-quick-reference.rst

```diff
@@ -0,0 +1,468 @@
+Architecture Quick Reference
+============================
+
+This is a quick reference guide to IoTempower's architecture and key concepts.
+
+For detailed information, see:
+
+- `Architecture Overview <architecture.rst>`_ - Complete system architecture
+- `Device Architecture <device-architecture.rst>`_ - Device framework details
+- `Deployment Process <deployment-process.rst>`_ - Build and deploy workflow
+
+
+Key Concepts
+------------
+
+Systems, Nodes, and Devices
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block::
+
+   IoT System (e.g., "my-home")
+   └── Nodes (e.g., "living-room", "bedroom")
+       └── Devices (e.g., "temperature", "motion", "led")
+
+**System**: A complete IoT installation (one WiFi network, one MQTT broker)
+
+**Node**: A microcontroller (ESP8266/ESP32) with connected devices
+
+**Device**: A sensor or actuator (temperature sensor, LED, button, etc.)
+
+
+Device Class Hierarchy
+~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block::
+
+   Device (base class)
+   ├── Input_Base → Input, Analog
+   ├── Output → PWM, RGB_Single, RGB_Strip_Bus
```

Comment:
output itself is also a device. PWM, RGB_Single, and RGB_Strip_Bus are not inheriting from the Output device class. PWM is on its own (just a device), and RGB_Single, and RGB_Strip_Bus are inheriting from RGB_Base. So better to list them directly under Device here, but maybe just after the Output device to avoid confusion, as we are talking about the Class hierarchy.